public/pages/DailyInsights/components/IndicesManagement.tsx

```diff
@@ -124,34 +181,6 @@ const fetchInsightsStatus = async () => {
   }
 };
 
-  const handleDataSourceChange = (dataSources: any[]) => {
```

Comment:
nit: for line 174 and 175 above, why doing it in http call way instead of calling it through router?